esx_service/utils/auth_data.py

```diff
@@ -1246,6 +1259,10 @@ def remove_tenant(self, tenant_id, remove_volumes):
             error_msg = self.__remove_volumes_for_tenant(tenant_id)
             if error_msg:
                 return error_msg
+        else:
+            error_msg = self.remove_symlink_for_tenant(tenant_id)
```

Comment:
Can this new function be used in __remove_volumes_for_tenant() vs. having the symlink removal code in two places once in this function and in __remove_volumes_for_tenant() as well.